app/utils/callback/lava_filters.py

```diff
@@ -37,7 +37,7 @@
 
 
 LAVA_SIGNAL_PATTERN = re.compile(
-    r'\<LAVA_SIGNAL_.+>')
+    r'\<LAVA_.+>')
```

Comment:
Seems worth adding `^` as discussed to only catch lines that start with `<LAVA_` since it should always be the case.

Reply:
fixed